CHANGELOG.md

```diff
@@ -1,9 +1,29 @@
-# Changelog
+# Change Log
 
-## 0.1.1 - 2023-03-09
+All notable changes to this project's source code will be documented in this file. Items under `Unreleased` is upcoming features that will be out in next version.
 
-- Fixed issue with default gems for older Ruby versions (#14)
+Please follow the recommendations outlined at [keepachangelog.com](https://keepachangelog.com). Please use the existing headings and styling as a guide, and add a link for the version diff at the bottom of the file. Also, please update the `Unreleased` link to compare to the latest release version.
 
-## 0.1.0 - 2023-02-19
+
+## Versions
+## [Unreleased]
+Changes since last non-beta release.
+
+*Please add entries here for your pull requests that are not yet released.*
+
+## [0.2.0] - 2023-03-10
+
+### Added
+
+- Add cleanup-old-images command [PR10](https://github.com/shakacode/heroku-to-control-plane/pull/10) by [Rafael Gomes](https://github.com/rafaelgomesxyz)
+- Add cleanup-stale-apps command [PR9](https://github.com/shakacode/heroku-to-control-plane/pull/9) by [Rafael Gomes](https://github.com/rafaelgomesxyz)
+
+### Fixed
+
+- Fixed issue with default gems for older Ruby versions [PR14](https://github.com/shakacode/heroku-to-control-plane/pull/14) by [Rafael Gomes](https://github.com/rafaelgomesxyz)
+
+## [0.1.0] - 2023-02-19
+
+### Added
```

Comment:
You are missing the link definitions at the bottom of the file. See other changelogs (view raw).

Reply:
Exactly.

At the moment of making this PR, we didn't have any tag on the project so I couldn't add version diff links at the bottom of the page. Now we have and I can add.